articles/logic-apps/logic-apps-limits-and-config.md

```diff
@@ -120,6 +120,21 @@ Following are limits for artifacts added to integration Account
 |Runtime endpoint tracking calls per 5 minutes |45,000|Can distribute workload across multiple accounts as needed|
 |Runtime endpoint blocking concurrent calls |~1,000|Decrease number of concurrent requests or reduce the duration as needed|
 
+Following are limits for count of artifacts added to Integration Account.
+
+Free pricing tier
+
+|Name|Limit|Notes|
+|----|----|----|
+|Agreements|10||
+|Other artifact types|25|The types include partners, schemas, certificates and maps. Each type can have up to the limit count of artifacts.|
+
+Standard pricing tier
+
+|Name|Limit|Notes|
+|----|----|----|
+|Any type of artifact|500|The types include agreements, partners, schemas, certificates and maps. Each type can have up to the limit count of artifacts.|
```

Comment:
|Any type of artifact|500|These types include agreements, partners, schemas, certificates, and maps. Each type can have up to the maximum number of artifacts.|